app/services/katello/pulp3/repository/yum.rb

```diff
@@ -4,6 +4,46 @@ module Katello
   module Pulp3
     class Repository
       class Yum < ::Katello::Pulp3::Repository
+        def self.api_client(smart_proxy)
+          PulpRpmClient::ApiClient.new(smart_proxy.pulp3_configuration(PulpRpmClient::Configuration))
+        end
+
+        def self.api_exception_class
+          PulpRpmClient::ApiError
+        end
+
+        def client_class
+          PulpRpmClient
+        end
+
+        def remote_class
+          PulpRpmClient::RpmRpmRemote
+        end
+
+        def remotes_api
+          self.class.remotes_api(smart_proxy)
+        end
+
+        def self.remotes_api(smart_proxy)
+          PulpRpmClient::RemotesRpmApi.new(api_client(smart_proxy))
+        end
+
+        def publication_class
+          PulpRpmClient::RpmRpmPublication
+        end
+
+        def publications_api
+          PulpRpmClient::PublicationsRpmApi.new(api_client)
+        end
+
+        def distribution_class
+          PulpRpmClient::RpmRpmDistribution
+        end
+
+        def self.distributions_api(smart_proxy)
+          PulpRpmClient::DistributionsRpmApi.new(api_client(smart_proxy))
+        end
```

Comment:
i think this might have been a rebase mistake?

Reply:
Yikes!